src/pages/UserHome.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import firebase from 'firebase';
+import {
+  Button,
+  Container,
+  Grid,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    paddingLeft: theme.spacing(6),
+  },
+  bannerRoot: {
+    backgroundColor: 'white',
+    marginTop: theme.spacing(4),
+    minHeight: '225px',
+    paddingLeft: theme.spacing(3),
+    paddingTop: theme.spacing(4),
+  },
+  browseButton: {
+    marginTop: theme.spacing(3),
+  },
+  addButton: {
+    textAlign: 'right',
+  },
+  scholarshipsAdded: {
+    marginTop: theme.spacing(6),
+  },
+}));
+
+export default function UserHome() {
+  const classes = useStyles();
+  const user = firebase.auth().currentUser;
+
+  function scholarshipLink() {
+    window.location.href = '/scholarships';
```

Comment:
This isn't how we've been doing in-site navigation. You can see what I mean by clicking the buttons you added here and noticing how the whole page gets refreshed.

You should use a react router [Link](https://reactrouter.com/web/api/Link). Similar to what was done [here](https://github.com/beyondhb1079/s4us/blob/0a46a5b743f939dda189dcf035d382aa6aabe1f9/src/components/HeaderNavMenu.jsx#L28) but instead of `MuiLink` you have a `Button`.